src/System.Runtime.Serialization.Xml/tests/SerializationTypes.cs

```diff
@@ -2711,4 +2711,21 @@ public class TypeWithTimeSpanProperty
 public class TypeWithByteProperty
 {
     public byte ByteProperty;
-}
\ No newline at end of file
+}
+
+[DataContract(Name = "TestLevelOne", Namespace = "")]
+public class TestLevelOne
```

Comment:
TestLevelOne is a verb phrase and more suitable for method name than class name, better use a descriptive noun phrase.